python/common/org/python/types/Int.java

```diff
@@ -474,6 +476,10 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
                 return (new org.python.types.Complex(this.value, 0)).__pow__(other, modulo);
             }
             return new org.python.types.Float(java.lang.Math.pow((double) this.value, other_val));
+        } else if (other instanceof org.python.types.Complex) { 
+            org.python.types.Complex cmplx_obj = new org.python.types.Complex((double) this.value, 0.0);
+            org.python.types.Complex other_cmplx_obj = (org.python.types.Complex) other;
+            return cmplx_obj.__pow__(other_cmplx_obj, null);
```

Comment:
The power checks are failing due to small rounding errors. For example:

```
- (0.7037573231697811+0.7104404479510611j)
?                  -
+ (0.703757323169781+0.7104404479510611j)
  
```

Any advice here?